tox.ini

```diff
@@ -1,10 +1,10 @@
 # content of: tox.ini , put in same dir as setup.py
 [tox]
-envlist = clean, style_check, py38, doctest
+envlist = clean, style_check, py310, doctest
 
 [gh-actions]
 python =
-    3.8: py38
+    3.10: py310
```

Comment:
Thanks for this great change. Can you add py310 but not remove py38 though? This project is supposed to be support python version >= 3.8, I'm worried change this directly may not be able to guarantee this anymore.

Reply:
I have updated the tox environments and also the version matrix on the github workflows; not sure if I've done it correctly, so happy to correct this further.